sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryOptions.java

```diff
@@ -85,6 +85,13 @@
 
   void setUseStorageWriteApi(Boolean value);
 
+  @Description(
+      "If set, then BigQueryIO.Write will default to using the approximate Storage Write API.")
```

Comment:
```suggestion
      "If set, then BigQueryIO.Write will use the Storage Write API with lower latency and relaxed consistency guarantees.")
```
Something like that?

Reply:
second half looks good. First half isn't quite right, since it will only default (i.e. it's the default if withMethod isn't called, otherwise this is ignored).

Reply:
gotcha. Makes sense. Feel free to adjust as you see fit.